infra/utils/fbf/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/utils/fbf
 
 go 1.22
```

Comment:
Do we need this if you're containing the toolchain to 1.23.4?

Reply:
Yeah - The toolchain is a recent requirement, it indicates what version of the GO binary last modified the file, not which version of GO is required.